app/models/storage/entry.rb

```diff
@@ -0,0 +1,34 @@
+class Storage::Entry < ApplicationRecord
+  belongs_to :account
+  belongs_to :board, optional: true
+  belongs_to :recordable, polymorphic: true, optional: true
+
+  # Accepts either objects or _id params (for after_destroy_commit snapshots)
+  def self.record!(delta:, operation:, account: nil, account_id: nil, board: nil, board_id: nil,
```

Comment:
I think we tend to avoid `!` in the methods unless to express a variation of another one with the same name (or for dangerous operations). I see you went with `!` in several methods. I am curious about the reason.

Reply:
Agree they're out of step. Old habits.